MykhailoTsviklinskyi/HW7/task3.py

```diff
@@ -0,0 +1,16 @@
+def calculate_characters_number(query: str)->dict:
```

Comment:
good job
look at this implementation 
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result